src/lib/ffi/ffi_pkey_algs.cpp

```diff
@@ -441,6 +459,24 @@ int botan_privkey_load_ecdsa(botan_privkey_t* key,
    }
 
 /* ElGamal specific operations */
+int botan_privkey_create_elgamal(botan_privkey_t* key, botan_rng_t rng_obj, size_t pbits)
+   {
+#if defined(BOTAN_HAS_ELGAMAL)
+
+    if(rng_obj == nullptr)
+      return BOTAN_FFI_ERROR_NULL_POINTER;
+
+    return ffi_guard_thunk(BOTAN_CURRENT_FUNCTION, [=]() -> int {
+      Botan::RandomNumberGenerator& rng = safe_get(rng_obj);
+      Botan::DL_Group group(rng, Botan::DL_Group::Strong, pbits);
```

Comment:
I've noticed that build for ARM fails because it takes long time to run tests. I think, it's because of the fact strong prime generation takes some time. I don't have immediate idea on how to improve performance here. So I'm wondering on either blocking the test or using Prime_Subgroup (ideally I would prefer strong primes, but from the other hand elgamal is not very popular anymore). What do you think?